src/arch/riscv/regs/vector.hh

```diff
@@ -47,10 +48,12 @@ namespace RiscvISA
 
 constexpr unsigned NumVecElemPerVecReg = 4;
 using VecElem = uint64_t;
-constexpr size_t vlenb = NumVecElemPerVecReg * sizeof(VecElem);
-constexpr size_t VLEN = vlenb * 8;
+constexpr size_t VLENB = NumVecElemPerVecReg * sizeof(VecElem);
```

Comment:
`vlenb` is intentionally lowercase in the spec, I think it's because the register names are lowercase. Maybe we should keep it lowercase.

Reply:
I thought it would be clearer that this was a global variable when it was referenced in other files. I'm not sure if Gem5 has a convention regarding this. All the other globals appeared to be upper case.

Example:

```
    for (size_t i = 0; i < VLENB; i++) {
        %(memacc_code)s;
    }
```

I realize I also changed this string be uppercase, which I shouldn't have:

```
    {CSR_VTYPE,        {"vtype" , MISCREG_VTYPE}},
    {CSR_VLENB,        {"VLENB" , MISCREG_VLENB}}
```

Should I change this back to lower case and keep the rest of the uppercase changes or should I change everything back to lowercase?

Reply:
Yes, I think change the register name back to lowercase and leave other parts as they are should be fine.

Reply:
For now I've just changed this back to lowercase:

```
    {CSR_VTYPE,        {"vtype" , MISCREG_VTYPE}},
    {CSR_VLENB,        {"vlenb" , MISCREG_VLENB}}
```